content/installation/java/standard/DebianRepository.md

```diff
@@ -18,5 +18,8 @@ echo "deb https://pkg.contrastsecurity.com/debian-public/ all contrast" | sudo t
 ```console
 sudo apt-get update && sudo apt-get install contrast-java-agent
 ```
+The above command will install the contrast.jar file in the /opt/contrast directory in your file system. If the directory is not available, it will create it automatically. 
```

Comment:
specifically, `/opt/contrast/contrast-agent.jar`. Let's spell out the whole path

```
/# dpkg -L contrast-java-agent
/usr
/usr/share
/usr/share/doc
/usr/share/doc/contrast-java-agent
/usr/share/doc/contrast-java-agent/copyright
/usr/share/doc/contrast-java-agent/changelog.gz
/opt
/opt/contrast
/opt/contrast/contrast-agent.jar
```

Reply:
> If the directory is not available, it will create it automatically. 

Do we need to call this out? I don't want to confuse anyone by adding unnecessary details. I think it's implied that the directories are created, because we never instructed the user to create the directory.

Reply:
it's fine, we don't need to mention it.